packages/lit-dev-content/site/tutorials/content/pointer-events/04.md

```diff
@@ -0,0 +1,20 @@
+Another powerful feature of Pointer Events is the ability to control the brower's native touch behavior with the use of the [`touch-action`](https://developer.mozilla.org/en-US/docs/Web/CSS/touch-action) css property.
+
+A pointer will fire events until the browser determines that a native behavior should occur, such as scrolling or zooming, at which point the browser will fire a final `pointercancel` event for the interaction.
+
+**Touch Action Settings**
+
+* `auto`
+  * This is the default setting. This means that the browser will fire events until scrolling, zooming, or browser specific behavior such as "double tap to zoom" occurs.
+* `none`
+  * With this setting, the browser will always fire events, and will prevent all touch behaviors like scrolling and zooming.
+* `pan-left`, `pan-right`, `pan-x`, `pan-up`, `pan-down`, and `pan-y`
+  * These settings will let the browser scroll in the specified direction.
+  * `pan-x` is shorthand for `pan-left pan-right`
+  * `pan-y` is shorthand for `pan-up pan-down`
+* `pinch-zoom`
+  * This setting will let the browser perform pinch to zoom behaviors, but not scrolling or "double tap to zoom".
+* `manipulation`
+  * This setting is equivalent to `pan-x pan-y pinch-zoom`, only disabling "double tap to zoom" and other browser-specific behaviors.
+
+Try using a touchscreen with this example to see how the different settings for `touch-action` control event firing and browser behavior.
```

Comment:
or use chrome dev tools' mobile emulation mode

Also again an appeal to point to sections of the code that would be relevant with a code snippet etc.